### ISSUE TITLE
Rename to gl-env-configuration

### ISSUE THREAD

**Issue by sandorfr** [geeklearningio/gl-ionic2-env-configuration#6]
It has no dependency to Ionic nor should it have. So we should rename it.

**Comment by mvidailhet** [comment]
You're totally right. But I dind't really have the time to check if it work for a regular Angular 2 website. If you're telling me it works for you, then I'll rename it !

**Comment by mvidailhet** [comment]
I already have a  gl-angular-configuration https://github.com/geeklearningio/gl-angular-configuration for angular 1.5 though...
